src/components/Wallet/ConfirmWallet.jsx

```diff
@@ -25,6 +27,11 @@ class WalletConfirmation extends React.Component {
           </TableHead>
           <TableBody>{this.renderConfirmationInfo()}</TableBody>
         </Table>
+        {startingAddressIndex > 0 && (
+          <Alert severity="info">
+            Setting Address Index set to {startingAddressIndex}
+          </Alert>
+        )}
```

Comment:
What do you think about this?

```diff
diff --git a/src/components/Wallet/ConfirmWallet.jsx b/src/components/Wallet/ConfirmWallet.jsx
index 53fd28d..93e905a 100644
--- a/src/components/Wallet/ConfirmWallet.jsx
+++ b/src/components/Wallet/ConfirmWallet.jsx
@@ -17,6 +17,12 @@ class WalletConfirmation extends React.Component {
     const { startingAddressIndex } = this.props;
     return (
       <Box>
+        {startingAddressIndex > 0 && (
+          <Alert severity="info">
+            Setting Address Index set to {startingAddressIndex}
+          </Alert>
+        )}
+
         <Table>
           <TableHead>
             <TableRow>
@@ -27,11 +33,6 @@ class WalletConfirmation extends React.Component {
           </TableHead>
           <TableBody>{this.renderConfirmationInfo()}</TableBody>
         </Table>
-        {startingAddressIndex > 0 && (
-          <Alert severity="info">
-            Setting Address Index set to {startingAddressIndex}
-          </Alert>
-        )}
       </Box>
     );
   };
```

Basically moving the alert up to the top of the box.

![gscreenshot_2020-07-06-074432](https://user-images.githubusercontent.com/4116405/86589917-8c4d6400-bf5c-11ea-8ba6-31236149459b.png)

Reply:
That's a typo, should be 'Starting Address Index'.
I agree on placement, making this change.